pipelines/release/docker/build_stack.groovy

```diff
@@ -48,11 +48,12 @@ notify.wrap {
   def gitRef         = dockerfile.git_ref
   def buildDir       = dockerfile.dir
   def dockerRepo     = dockerRegistry.repo
-  def dockerTag      = "7-stack-lsst_distrib-${eupsTag}"
+  def dockerTag      = "9-stack-lsst_distrib-${eupsTag}"
```

Comment:
This is the one thing that will require some pre-announcement to downstream consumers before we actually do it.  And if we're going to change it, I thought we were going to change to something more sensible?